src/pages/index.js

```diff
@@ -52,51 +57,51 @@ const LEAD = `Easier, scalable and zero-maintenance. With security, scorecards a
-        'Roadie is SOC2 Type 2 Certified and uses ephemeral environments for your Scaffolder actions, a common source of risk for Open Source adopters.',
+        'Say goodbye to endless searching with Roadie’s software catalog. Discover and ingest all your organization’s microservices, APIs, resources, and teams into one centralized place. You’ll never have to wonder what exists or who owns it again.',
+        'Integrate documentation with code to create a single source of truth for every entity in your ecosystem - forget tribal knowledge. Easily keep everything up-to-date, discoverable, and accessible, reducing errors and ensuring everyone is on the same page.'
       ],
     },
     {
-      title: 'Get all the OSS features, simplified',
-      description: 'Roadie brings no-code management to Backstage',
+      title: 'Move Fast and (Don’t) Break Things',
+      description: 'Deploy fast, securely and consistently with Templates',
       illustration: {
-        png: NoCodeImg,
+        png: TemplatesImg,
         alt: '',
       },
       paragraphs: [
-        'With a vibrant community of contributors, new Backstage plugins and features are popping up regularly. Roadie brings all these features, while smoothening out rough edges like GitHub rate limits.',
-        'Roadie enables plugins and integrations through a UI, which also lets you manage access and permissions. Roadie also provides advanced debugging capabilities to make it easy to navigate the unexpected, if it happens.',
+        'Service templates are blueprints for golden paths to production - they enforce security and compliance with baked-in best practices, so developers can launch services consistently and confidently.',
+        'With service templates your team can deploy faster and autonomously without bottlenecks, so developers can get code and infrastructure live faster with no need to wait for manual approvals or ops intervention.'
       ],
     },
     {
-      title: 'Make your Developer Portal truly yours',
-      description: 'Roadie lets you install private plugins and renderers',
+      title: 'Spot Code That Doesn’t Meet Standards',
+      description: 'Identify where code falls short of standards. Fix it fast',
       illustration: {
-        png: CustomPluginsImg,
+        png: TechInsightsImg,
         alt: '',
       },
       paragraphs: [
-        'Your Developer Portal will only be successful if it’s tailored to the way your developers work. Roadie lets you bring your own Backstage plugins so you can integrate internal systems into your Developer Portal.',
-        'Roadie also lets you bring your own API documentation renderer so your docs are presented exactly as you want them inside Backstage.',
+        'Set engineering standards and create a culture of accountability with software checks that automatically ensure software meets expectations for security, compliance, and deployment. Roadie checks catch quality issues before they become problems.',
```

Comment:
Wondering if we should put capitalization or italics on "Roadie Checks" for sentence flow.

```suggestion
        'Set engineering standards and create a culture of accountability with software checks that automatically ensure software meets expectations for security, compliance, and deployment. _Roadie Checks_ catch quality issues before they become problems.',
```